arkane/encorr/bac.py

```diff
@@ -107,14 +110,14 @@ def execute(self, output_directory: str = None, plot: bool = False, jobnum: int
         logging.info(f'Running BAC job {jobnum}')
         self.bac.fit(**self.kwargs)
 
-        if output_directory is not None:
+        if output_directory is not None and self.crossval_n_folds == 1:
             os.makedirs(output_directory, exist_ok=True)
```

Comment:
@kspieks can you explain this change really quick?

Reply:
Colin had added a [change](https://github.com/ReactionMechanismGenerator/RMG-Py/pull/1968/commits/9e3a19484c2458f7196d06a30336ff342aa4630d#diff-a4352d67c1b492a6db2aa46f53d796701efb51a956c2a119b3928f6b8088d730R117) to only write to the database if not doing cross-validation, since we only want to add values that have been trained on the entire dataset.

Extending this line of thinking, I thought it was potentially misleading to print `output.py` with BAC values as well as the summary csv for BACs that were only trained on the last fold from cross-validation i.e. they were not trained on the entire reference set. Do follow up if this doesn't make sense.